### PR TITLE
feat(@vtmn/css): add badge component

### PR DESCRIPTION
**Pull request by GaspardMathon**
## Pull Request checklist

🚨 Please review the [guidelines for contributing](../blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Don't request your main!
- [x] Make sure you are making a pull request against the **main branch** (left side).
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

## Description

- Add badge component
- Add version with icon font and with a label

## Does this introduce a breaking change?

- No

❤️ Thank you!

### CODE REVIEW

File: packages/sources/css/src/components/button/src/index.css [hunk 1]
```diff
@@ -198,7 +198,64 @@
...
+
+.vtmn-btn_size--large.vtmn-btn_icon--alone span {
+  font-size: var(--vtmn-typo_title-5-font-size);
+}
+
+.vtmn-btn_size--large.vtmn-btn_icon--along span {
+  padding-right: rem(12px);
+}
+
+/* .vtmn-btn-icon {
```
Comment (lauthieb) [inline packages/sources/css/src/components/button/src/index.css:258]: Why in this PR?

File: packages/sources/css/src/components/badge/src/index.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  display: inline;
+}
+
+.vtmn-badge:focus-visible {
+  outline: none;
+  box-shadow: 0 0 0 rem(4px) var(--vtmn-color_white),
+    0 0 0 rem(6px) var(--vtmn-color_black);
+}
+
+.vtmn-badge span[class^='vtmx-'] {
```
Comment (lauthieb) [inline packages/sources/css/src/components/badge/src/index.css:18]: Badge should also work with `img` and `svg`for CSS styles

File: packages/showcases/css/stories/components/button/examples/with-icon.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    Button
+  </button>
+</div>
+
+<div class="block">
+  <button class="vtmn-btn vtmn-btn_size--large vtmn-btn_icon--along">
+    <span class="vtmx-search-line"></span>
+    Button
+  </button>
+</div>
```
Comment (lauthieb) [inline packages/showcases/css/stories/components/button/examples/with-icon.html:38]: Again, why in this PR?

File: packages/showcases/css/stories/components/button/examples/overview.html [hunk 4]
```diff
@@ -1,3 +1,16 @@
...
+    <span class="vtmx-search-line"></span>
+  </button>
+</div>
+
+<div class="block">
+  <button class="vtmn-btn vtmn-btn_icon--along">
+    <span class="vtmx-search-line"></span>
+    Button
+  </button>
+</div>
```
Comment (lauthieb) [inline packages/showcases/css/stories/components/button/examples/overview.html:16]: Again, why in this PR?

File: packages/showcases/css/stories/components/button/button.stories.js [hunk 5]
```diff
@@ -30,6 +31,15 @@ Overview.parameters = {
...
 
+export const WithIcon = () => withIcon;
+WithIcon.parameters = {
+  docs: {
+    source: {
+      code: withIcon,
+    },
+  },
+};
+
```
Comment (lauthieb) [inline packages/showcases/css/stories/components/button/button.stories.js:42]: Again, why in this PR?

File: packages/showcases/css/stories/components/badge/badge.stories.js [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+};
+
+export const Default = () => _default;
+Default.parameters = {
+  docs: {
+    source: {
+      code: _default,
+    },
+  },
+};
```
Comment (lauthieb) [inline packages/showcases/css/stories/components/badge/badge.stories.js:38]: Are default and brand the same? If yes, you can delete this "Default" story as I did on other components

**Review by lauthieb — CHANGES_REQUESTED** [review]
Thanks @GaspardMathon, few changes!

File: packages/showcases/css/stories/components/badge/badge.stories.js (hunk 6 above)
Comment (GaspardMathon) [reply packages/showcases/css/stories/components/badge/badge.stories.js:38]: No they have different properties, the color for instance.

File: packages/sources/css/src/components/button/src/index.css (hunk 1 above)
Comment (GaspardMathon) [reply packages/sources/css/src/components/button/src/index.css]: I deleted all commits on button component

File: packages/showcases/css/stories/components/button/examples/with-icon.html (hunk 3 above)
Comment (GaspardMathon) [reply packages/showcases/css/stories/components/button/examples/with-icon.html]: Deleted

File: packages/showcases/css/stories/components/button/examples/overview.html (hunk 4 above)
Comment (GaspardMathon) [reply packages/showcases/css/stories/components/button/examples/overview.html]: Deleted

File: packages/showcases/css/stories/components/button/button.stories.js (hunk 5 above)
Comment (GaspardMathon) [reply packages/showcases/css/stories/components/button/button.stories.js]: Deleted

File: packages/sources/css/src/components/badge/src/index.css (hunk 2 above)
Comment (GaspardMathon) [reply packages/sources/css/src/components/badge/src/index.css:18]: From now, badge works with svg iconq

**Comment by lauthieb** [comment]
As we spoke together, for me a badge is just a little circle. 
But maybe we can help developers to have their icon badges & label badges easily.
I'm available to speak with designers & to pair-prog together!

**Pull request merged**